awslabs/amazon-kinesis-video-streams-ice
# Created Unit-Test Env with basic unit test

*Description of changes:*
This PR adds unit-tests structure for ICE along-side with updating of `README.md.` 
Structure of `IceCredentials` was modified for compatibility. 

*Test Steps:*

```

cmake -S test/unit-test -B build/ -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Debug -DBUILD_CLONE_SUBMODULES=ON -DCMAKE_C_FLAGS='--coverage -Wall -Wextra -Werror -DNDEBUG -DLIBRARY_LOG_LEVEL=LOG_DEBUG'

cd build && make coverage

```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
